demisto_sdk/commands/generate_outputs/generate_descriptions/tests/generate_descriptions_test.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+import requests_mock
+
+from demisto_sdk.commands.common.legacy_git_tools import git_path
+from demisto_sdk.commands.common.tools import get_json, get_yaml
+
+# Test data files
+
+
+FAKE_INTEGRATION_YML = open(
+    f'{git_path()}/demisto_sdk/commands/generate_outputs/generate_descriptions/tests/test_data/input_integration.yml', 'r').read()
+FAKE_OUTPUT_AI21 = get_json(
+    f'{git_path()}/demisto_sdk/commands/generate_outputs/generate_descriptions/tests/test_data/ai21_response.json')
+LOGPROB_INPUT_AI21 = get_json(
+    f'{git_path()}/demisto_sdk/commands/generate_outputs/generate_descriptions/tests/test_data/logprob_res.json')
+
```

Comment:
please add a flow test that checks that an output that resets itself and was approved by the year user will be preposed the same description.